addons/web/static/src/js/views/form_common.js

```diff
@@ -1033,6 +1033,9 @@ var SelectCreatePopup = AbstractFormPopup.extend({
                 });
                 $cbutton = self.$buttonpane.find(".oe_selectcreatepopup-search-create");
                 $cbutton.click(function() {
+                    if ($(".fa-caret-up").is(":visible")){
```

Comment:
No, $(".fa-caret-up").is(":visible") means select all .fa-catet-up from whole DOM and if it visible then toggle bottons